neurodamus/stimulus_manager.py

```diff
@@ -76,7 +95,7 @@ class BaseStim:
 
     IsNoise = False
 
-    def __init__(self, _target, stim_info: dict, _cell_manager):
+    def __init__(self, _tpoints, stim_info: dict, _cell_manager):
```

Comment:
I assume `tpoints` means `target_points`? I could only guess that because the previous name was `target`.
If they are going to be renamed, is there a reason not to call them `locations`?

Reply:
> tpoints are probably taget points. They are named everywhere as TPoints and stuffed in a list. A TPointList.

Hmm, good to know.  It's too bad that knowledge is gone, because it feels relevant.  Thanks for the explanation.

>  I can rename then as locations

No, that's fine, leave it as it is.

Reply:
> ah c'mon the knowledge is not gone. I just explained (by re-reading the code) and Weina did this (I think)

Perhaps gone is too harsh; but it requires guessing, at least.

Reply:
>I assume tpoints means target_points?

Yes, I think we could name that. It refers to the points of a target.